articles/lets-learn-about-osi-layer-model/index.md

```diff
@@ -0,0 +1,133 @@
+- Addressing: A Network layer adds the source and destination address to the header of the frame. Addressing is used to identify the device on the internet.
+- Routing: Routing is the major component of the network layer, and it determines the best optimal path out of the multiple paths from source to destination.
+- Packetizing: A Network Layer receives the packets from the upper layer and converts them into packets. This process is known as Packetizing. It is achieved by internet protocol (IP).
+
+#### Transport Layer
+- It transmits data using transmission protocols including TCP and UDP. 
+- The transport layer takes data transferred in the session layer and breaks it into “segments” on the transmitting end. 
+- It's also responsible for reassembling the segments on the receiving end, turning them back into data that can be used by the session layer
+
+
+#### Session Layer
+When two devices, computers, or servers need to “speak” with one another, a session needs to be created, and this is done at the Session Layer. 
+- It creates communication channels called sessions between nodes. 
+- It also set checkpoints during a data transfer—if the session is interrupted, devices can resume data transfer from the last checkpoint.  
+- It maintains connections and responsible for controlling ports and sessions.
+
+#### Presentation Layer
+The Presentation Layer represents the area that is independent of data representation at the application layer. 
+- It represents the preparation or translation of application format to network format, or from network formatting to application format. In other words, the layer “presents” data for the application or the network. It ensures that data is in a usable format and is where data encryption occurs.
+
+##### The function of Presentation Layers:
+- Character code translation from ASCII to EBCDIC.
+- Data compression: This allows to reduce the number of bits that need to be transmitted on the network.
+- Data encryption: Helps you to encrypt data for security purposes — for example, password encryption.
+- It provides a user interface and support for services like email and file transfer.
+
+
+### Application Layer
+
+- The application layer interacts with an application program that is closest to the end-user. 
+- It allows users to interact with other software applications. 
+- The application layer interacts with software applications to implement a communicating component. 
+
+##### The function of the Application Layers are:
+- Application-layer helps you to identify communication partners, determining resource availability, and synchronizing communication.
+- It allows users to log on to a remote host and provides various e-mail services.
+- This application offers distributed database sources and access to global information about various objects and service
+ 
+Quick Way to memorize the OSI Model Layer concept - A Penguin Said That Nobody Drinks Pepsi
+
+ 
```

Comment:
Add a Conclusion section, and conclude the article gracefully.

Reply:
@lalith1403 I have added it